docs/UserGuide.adoc

```diff
@@ -14,6 +14,18 @@ endif::[]
 
 By: `Team SE-EDU`      Since: `Jun 2016`      Licence: `MIT`
 
+== Introduction
+Thank you for downloading **UniFy**! We're glad that you've chosen us to help you manage your hectic university life.
+
+**UniFy** is a personal assistant to your everyday life in NUS. It helps you maintain your numerous groups,
+tells you when your friends are free for lunch and find common time for project meetings. It even helps you find directions to
+wherever your friends might be!
```

Comment:
'wherever your friends might be'? Is it related to the GPS thing which we don't intend to implement?   :)

Reply:
This refers to their address which is already implemented in the Person field :)